packages/frontend/src/components/InputCreator/index.tsx

```diff
@@ -67,20 +66,14 @@ export default function InputCreator(props: InputCreatorProps): JSX.Element {
       <ControlledAutocomplete
         name={computedName}
         dependsOn={dependsOn}
-        fullWidth
-        disablePortal
         required={required}
         freeSolo={schema.allowArbitrary}
         options={preparedOptions}
-        renderInput={(params) => (
-          <MuiTextField placeholder={placeholder} {...params} />
-        )}
         defaultValue={value as string}
         description={description}
         loading={loading}
         // if schema source is defined, dynamic data is supported
         onRefresh={schema.source ? () => refetch() : undefined}
-        disabled={disabled}
```

Comment:
eh why leave this out?

Reply:
cus i didnt see any usage in the autocomplete oops, may need a sanity check haha